pkgs/applications/networking/instant-messengers/signald/default.nix

```
@@ -0,0 +1,87 @@
+{ lib, stdenv, fetchurl, fetchgit, jre, coreutils, gradle_6, git, perl
+, makeWrapper }:
+
+let
+  pname = "signald";
+
+  version = "0.13.1";
+  src = fetchgit {
```

```suggestion
  version = "0.13.1";

  src = fetchgit {
```

Also please use fetchFromGitLab.

---

This package uses the .git directory sadly. I'll make a comment